### DISCUSSION TITLE
Disable transactions for a single migration

### DISCUSSION THREAD

**Discussion by captain-http** [prisma/prisma#10601] [Q&A] [answered]
**Is it possible to disable transactions for a particular database migration?** I want to build concurrently an index in Postgres, but that can't be done inside a transaction.

```
Database error code: 25001

Database error:
ERROR: CREATE INDEX CONCURRENTLY cannot run inside a transaction block
```

**Comment by janpio** [comment]
Discussion in Slack: https://prisma.slack.com/archives/CBFFY3066/p1638820809041100

**Reply by ryanmt** [reply, marked as answer]
For future reference, capturing this out of slack would be very helpful for those who aren't in your slack, or if you ever lose history.

TL;DR; after joining slack: TBD and this issue presents workaround options as well as makes a case for supporting concurrent index creation in prisma itself (rather than a way to avoid wrapping the migration in a transaction).  https://github.com/prisma/prisma/issues/9154

**Comment by jeffmhastings** [comment]
It works if you split each `CREATE INDEX CONCURRENTLY` statements into their own separate migration. Prisma does not seem to create the transaction when the migration only contains a single statement.
